openslides/poll/models.py

```diff
@@ -70,6 +70,7 @@ def print_weight(self, raw=False):
 PERCENT_BASE_CHOICES = (
     {'value': 'WITHOUT_INVALID', 'display_name': ugettext_lazy('Only all valid votes')},
     {'value': 'WITH_INVALID', 'display_name': ugettext_lazy('All votes cast (including invalid votes)')},
+    {'value': 'WITHOUT_ABSTAIN', 'display_name':  ugettext_lazy('Only YES and NO values')},
```

Comment:
use better: "Only Yes and No votes" for the new option.

Change the string "Only all valid votes" to "All valid votes (Yes/No/Abstain)". This should better explain the difference between the options.

And please change the order (descending):
* "Only Yes and No votes"
* "All valid votes (Yes/No/Abstain)"
* "All votes cast (including invalid votes)"
* "Disabled (no percents)"